package.json

```diff
@@ -17,28 +17,30 @@
     }
   ],
   "dependencies": {
-    "es6-promise": "^1.0.0",
-    "arraybuffer-slice": "~0.1.2",
-    "es5-shim": "^4.0.0"
   },
   "peerDependencies": {
     "freedom": "^0.6.11"
   },
   "devDependencies": {
-    "freedom": "^0.6.4",
+    "es6-promise": "^1.0.0",
+    "arraybuffer-slice": "~0.1.2",
+    "freedom": "^0.6.11",
     "grunt": "~0.4.5",
-    "grunt-contrib-jasmine": "~0.7.0",
+    "grunt-contrib-jasmine": "~0.8.0",
     "grunt-contrib-clean": "~0.6.0",
     "grunt-contrib-coffee": "~0.12.0",
-    "grunt-contrib-copy": "~0.6.0",
+    "grunt-contrib-copy": "~0.7.0",
     "grunt-contrib-symlink": "~0.3.0",
-    "grunt-contrib-uglify": "~0.6.0",
+    "grunt-contrib-uglify": "~0.7.0",
     "grunt-contrib-watch": "~0.6.1",
-    "grunt-ts": "^1.11",
-    "typescript": "^1.1.0-1"
+    "grunt-browserify": "^3.0.1",
+    "grunt-ts": "^2.0.1",
+    "grunt-tsd": "^0.1.0",
+    "tsify": "^0.7.1",
+    "typescript": "^1.3.0"
   },
   "scripts": {
-    "test": "grunt test",
-    "prepublish": "grunt test"
+    "test": "grunt unit_tests",
+    "postinstall": "grunt setup"
```

Comment:
i think this doesn't do what you want - i believe it runs every time your module is installed, even as a dependency of something else - in which case the devDependencies won't exist, so grunt won't be able to run setup for you.

see: https://docs.npmjs.com/misc/scripts#note-install-scripts-are-an-antipattern